modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -1111,12 +1118,15 @@ POSIX_Init ( void *argument __attribute__((unused)))
     {
         unsigned int p;
         if (epicsThreadHighestPriorityLevelBelow(epicsThreadPriorityScanLow, &p)
-                                            == epicsThreadBooleanStatusSuccess)
+                                            != epicsThreadBooleanStatusSuccess)
```

Comment:
Why call epicsThreadHighestPriorityLevelBelow at all?  The result `p` will always be an EPICS priority.  I think the mapping below should always be used starting with epicsThreadPriorityScanLow.